make hostpath failed

follow readme:
```shell
$ make hostpath
make: *** No rule to make target 'hostpath'.  Stop.
```

when i use `make` build success，Is the readme wrong?

> platform:  Mac And Linux

--- a/README.md
+++ b/README.md
@@ -24,7 +24,7 @@ The following examples assume that the CSI hostpath driver has been deployed and
 If you want to build the driver yourself, you can do so with the following command from the root directory:
 
 ```shell
-make hostpath
+make
 ```
 
 ## Community, discussion, contribution, and support

--- a/pkg/hostpath/README.md
+++ b/pkg/hostpath/README.md
@@ -4,12 +4,12 @@
 
 ### Build hostpathplugin
 ```
-$ make hostpath
+$ make
 ```
 
 ### Start Hostpath driver
 ```
-$ sudo ./_output/hostpathplugin --endpoint tcp://127.0.0.1:10000 --nodeid CSINode -v=5
+$ sudo ./bin/hostpathplugin --endpoint tcp://127.0.0.1:10000 --nodeid CSINode -v=5
 ```
 
 ### Test using csc

